libats/src/main/scala/com/advancedtelematic/libats/data/PaginationResult.scala

```diff
@@ -0,0 +1,13 @@
+package com.advancedtelematic.libats.data
+
+final case class PaginationResult[A](total: Long, limit: Long, offset: Long, values: Seq[A]) {
+  def map[B](f: A => B): PaginationResult[B] = this.copy(values = values.map(f))
```

Comment:
This looks useful, is there an implicit override of the default map happening here?